homecode-pipeline/homecode-pipeline.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+import kfp
+
+search_path = os.path.dirname(os.path.abspath(__file__)) + "/../components"
+component_store = kfp.components.ComponentStore(local_search_paths=[search_path])
+
+dkube_training_op = component_store.load_component("training")
+
+username = (
+    os.getenv("username") if os.getenv("username") != None else os.getenv("USERNAME")
+)
+
+
+@kfp.dsl.pipeline(name="homedir-pl", description="code from home dir")
+def homedircode_pipeline(
+    code, dataset, model, dataset_mount_path, model_mount_path, token
+):
+
+    run_script = (
+        "python /mnt/dkube/home/"
+        + str(username)
+        + "/workspace/"
+        + str(code)
+        + "/mnist/train.py"
+    )
+
+    train = dkube_training_op(
+        auth_token=str(token),
+        container='{"image":"ocdr/dkube-datascience-tf-cpu:v2.0.0"}',
+        framework="custom",
+        version="dummy",
+        run_script=run_script,
+        datasets=json.dumps([str(dataset)]),
+        outputs=json.dumps([str(model)]),
+        input_dataset_mounts=json.dumps([str(dataset_mount_path)]),
```

Comment:
What this pipeline ? Maybe we should add README.md file

Reply:
this pipeline is basically using code from homedir, for the training op we do not pass code details, to do that we use custom framework, since for other options passing code details as argument is necessary.